submissions/Vas-P/popup/css/style.css

```diff
@@ -53,23 +61,29 @@ nav{
 	height: 20px;
 }
 
-.popup-button{
+.popup_button{
 	cursor: pointer;
+	display: flex;
+	border: 1px solid white;
+	background: url(../img/popup-button.png) center no-repeat;
+	width: 30px;
+	height: 30px;
 }
 
-.popup-button::before{
-	opacity: 0;
+.nav_input:focus+.popup_button{
+	border: 1px solid #989898;
+	border-radius: 2px;
+}
+
+.nav_arrow{
```

Comment:
Popup triangle still present in HTML markup. remove it from markup, use pseudoelements `::before` and `::after` of `.hidden` block.

Reply:
Ok, i'll try it, thank you.